### PR TITLE
Exercises

### PR DESCRIPTION
**Pull request by vinayshah105**

### CODE REVIEW

**Comment by vinayshah105** [comment]
Please Check exercise

File: functions/exercise/1_convertMinutestoSec.js [hunk 1]
```diff
@@ -3,3 +3,7 @@
 // Expected return value : The number of seconds is 600
 
 
+function convertMinuteToSeconds(min){
+    sec = min * 60;
+}
```
Comment (divyacnr03) [inline functions/exercise/1_convertMinutestoSec.js:8]: Sec needs to returned

File: functions/exercise/5_findlargestInNumberArray.js [hunk 2]
```diff
@@ -8,4 +8,16 @@
 
 
 // Sample function : findLargestNumber([])
-// Expected return value : null
\ No newline at end of file
+// Expected return value : null
+
+var number = [5, 6, 12, 35, -12]
+function findLargestNumber([number]){
```
Comment (divyacnr03) [inline functions/exercise/5_findlargestInNumberArray.js:14]: YOu dont have to have number in []

File: functions/exercise/3_implementAnd.js [hunk 3]
```diff
@@ -13,4 +13,45 @@
...
+
+function andOperator(x,y)
+{
+var x;
+var y;
+
+    if(x == 0){
+        if(y == 0){
+            console.log("false")
+        }
```
Comment (divyacnr03) [inline functions/exercise/3_implementAnd.js:27]: Once x == 0 , no matter if y is 0 or not the output will be false, so you can skip the line 25

File: functions/exercise/3_implementAnd.js [hunk 4]
```diff
@@ -13,4 +13,45 @@
...
+        if(y == 1){
+            console.log("True")
+        }
+    }
+    if(x == null){
+        if(y == 1){
+            console.log("false")
+        }
+    }
+    if(x == 1){
```
Comment (divyacnr03) [inline functions/exercise/3_implementAnd.js:51]: Try using the else if conditions, that will help you avoid unwanted conditions

**Comment by divyacnr03** [comment]
Added my comments in individula files

**Pull request closed without merging**